### ISSUE TITLE
Unable to run Hurl on W10

### ISSUE THREAD

**Issue by cwichel** [U-C-S/Hurl#42]
Hi! 

I'm having issues with running Hurl...

My system: 
- i7-8700K, GeForce GTX 1070 Ti, 16GB RAM
- Microsoft Windows 10 Pro (10.0.19044)

The installation process: 
1. Downloaded and installed .NET 6 Desktop Runtime x64.
2. Installed Hurl from release v0.6.1 on default folder. 

Issue: Tried to run Hurl but nothing happened: 
- When executing Hurl from the exe file: No window is open, the user settings file is not generated on %APPDATA%.
- Hurl is listed on default browsers (and was selected) but opening links don't produce any outcome. 

Is there any step missing in the installation process? Is any other dependency/library required?

**Comment by U-C-S** [comment]
Thanks for reporting..... I haven't tested Hurl with Windows 10 properly. For now, I am assuming the App just crashed with some Error that's not handled properly. 

I'll look into it in the coming days

**Comment by hanjo** [comment]
Hi - I'm having the same issue on Windows 11 21H2 (Build 22000.675). Application launches and immediatly crashes.

Issue is with 0.6.1 only, 0.6 works fine.

**Comment by U-C-S** [comment]
> Issue is with 0.6.1 only, 0.6 works fine

@hanjo That's a good info and simplifies catching it. 

I'll start looking into this in next week.

**Comment by U-C-S** [comment]
Found the issue: 
![image](https://user-images.githubusercontent.com/50218121/171561765-493d2efa-484d-4193-9350-4ecfed30ea05.png)

# Workaround
For now, Workaround is to " Manually create a directory `Hurl` under `...../AppData/Roaming` ". So, Hurl can create the `UserSettings.json` file in it. 


### dev stuff
I handled this case pretty well in the code. But seems the code isn't getting called. 
https://github.com/U-C-S/Hurl/blob/53b721767315eeaa39aefc14a6c1c8ff782fcaef/App/Hurl.SharedLibraries/Services/SettingsFile.cs#L35

cc @hanjo , @cwichel

**Comment by U-C-S** [comment]
@hanjo Thanks once again... I am able reproduce this. 
Discovered that this part of the issue caused due to poor handling of #39 i.e. I forgot to make the fields optional.

For now, add the following to the `UserSettings.json` like seen in https://github.com/U-C-S/Hurl/discussions/40#discussioncomment-2613539

```json
  "AppSettings": {
    "DisableAcrylic": false,
    "BackgroundRGB": [ 51, 51, 51 ]
  },
```

**Comment by hanjo** [comment]
Thanks, I can confirm that 0.6.1 is running fine (i.e. not crashing on startup) after adding the above to the config file.

**Comment by cwichel** [comment]
I can confirm the same as @hanjo

**Comment by U-C-S** [comment]
Fixed this in v0.6.2
